dashboard/engines/marketing/config/routes.rb

```diff
@@ -0,0 +1,5 @@
+Marketing::Engine.routes.draw do
+  scope ':locale' do
```

Comment:
I forgot to mention that the standards V-Team also wanted to read the locale from the session via the normal locale retrieval mechanisms rather than having the API be stateless.